roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -0,0 +1,37 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
+
+location:
+  source_directories:
+  - {{ matrix_backup_borg_source }}
+  repositories:
+  - {{ matrix_backup_borg_repository }}
+  one_file_system: true
+  exclude_patterns:
```

Comment:
Perhaps: `exclude_patterns: {{ matrix_backup_borg_exclude_patterns|to_json }}` is better and safer:

- using `|to_json` to spew out the whole list in properly escaped form. Even if the list is empty (`matrix_backup_borg_exclude_patterns: []`), `|to_json` will still do the right thing. The `for` loop will not (although YAML may be crazy enough to still take an unterminated `exclude_patterns:` to be valid syntax..)